eyeshade/controllers/publishers.js

```diff
@@ -803,12 +941,15 @@ module.exports.initialize = async (debug, runtime) => {
      // v2 and later
         altcurrency: '',
         probi: bson.Decimal128.POSITIVE_ZERO,
+        currency: '',
+        amount: bson.Decimal128.POSITIVE_ZERO,
```

Comment:
we could probably get away with a regular number here instead of Decimal128 since this value is approx / for display. either is fine though

Reply:
sorry, "bump"?

perhaps. i just prefer these things to be consistent....

Reply:
bump was on the earlier comment to log to sentry if `probi.lessThan(0)` - not sure why it didn't thread correctly. no worries, sgtm

Reply:
that may not be an exception. with the bitcoin proof-of-concept, there was a one-time "thing" where i ran settlement twice. it's OK for probi to be zero in cases like that.

Reply:
Sorry I'm confused. Is the probi for publishers who went through double settlement negative or did it get zeroed? (I'm assuming it's okay to be zero means it's okay to zero here.) I still think it's a good idea to capture the case where it is negative. Obviously we'll get some events due to the double settlement, but over time assuming continued contributions we should expect all to go positive -> so an uptick in this event would be a red flag

Reply:
i've added a call to `runtime.captureException`